action.yml

```
@@ -26,10 +32,14 @@ runs:
         # Run the Julia command
         "${julia_cmd[@]}"
       shell: bash
-
     - run: |
         # The Julia command that will be executed
-        julia_cmd=( julia --color=yes --project=docs/ docs/make.jl )
+        julia_cmd=( julia --color=yes --code-coverage --project=docs/ -e '
+            push!(LOAD_PATH, "@docs-logger-env") # access GitHubActions.jl
+            import Logging, GitHubActions
+            Logging.global_logger(GitHubActions.GitHubActionsLogger())
+            pop!(LOAD_PATH)
```

Not sure this works in all Julia 1.X but maybe?
```suggestion
            @eval Module() begin
                push!(LOAD_PATH, "@docs-logger-env") # access GitHubActions.jl
                import Logging, GitHubActions
                Logging.global_logger(GitHubActions.GitHubActionsLogger())
                pop!(LOAD_PATH)
            end
```

---

I can give it a try

---

seems to work! https://github.com/ericphanson/VisualStringDistances.jl/runs/3726504869?check_suite_focus=true#step:5:186

---

works on 1.0! https://github.com/ericphanson/VisualStringDistances.jl/runs/3726621915